api/src/main/resources/org/openmrs/api/db/hibernate/ObsReferenceRange.hbm.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+	"-//Hibernate/Hibernate Mapping DTD 3.1//EN"
+	"http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.openmrs">
+
+	<class name="ObsReferenceRange" table="obs_reference_range" batch-size="25">
+
+		<cache usage="read-write"/>
+
+		<id name="obsReferenceRangeId" type="java.lang.Integer" column="obs_reference_range_id" unsaved-value="undefined">
```

Comment:
Is there any reason why you used a hibernate xml mapping file for this instead of annotations?

Reply:
But we cannot use both xml mapping files and annotations for the same entity. 😊